apps/prairielearn/src/pages/userSettings/userSettings.ts

```diff
@@ -45,6 +45,12 @@ router.get(
 
     const purchases = isEnterprise() ? await getPurchasesForUser(authn_user.user_id) : [];
 
+    const mode = await sqldb.callRow(
+      'ip_to_mode',
+      [req.ip, res.locals.req_date, authn_user.user_id],
+      ModeSchema,
+    );
```

Comment:
That's intentional, in part because all the handling of that is currently centralized in `authzCourseOrInstance`, which isn't applicable here as there isn't a specific course or course instance in play.